src/Microsoft.Health.Core/ActionTimer.cs

```diff
@@ -30,10 +30,7 @@ public ActionTimer(ILogger logger, string componentName)
             _stopwatch.Start();
         }
 
-        public TimeSpan GetElapsedTime()
-        {
-            return _stopwatch.Elapsed;
-        }
+        public TimeSpan ElapsedTime => _stopwatch.Elapsed;
```

Comment:
This is a very breaking change, but I am also making other breaking changes here. So I thought I'd include it (the analyzer also flags this as something that should be a property)